### ISSUE TITLE
Add another example in the section Foreign key about potential error

### ISSUE THREAD

**Issue by shenshan** [datajoint/datajoint-docs-original#93]
```
-> A
---
(x1, y1) -> B(x, y)
```
If `B` contains another attribute z in the primary key, and `z` is not part of the primary key attributes in `A`, it will throw an error. To include this extra attribute `z`, we would have to say `(x1, y1, z) -> B(x, y, z)`, or rename it such as`(x1, y1, z1) -> B(x, y, z)`

**Comment by dimitri-yatsenko** [comment]
Isn't this a problem in `datajoint-python`?  `z` should be included. 

Let's not worry about this because this will get fixed as part of https://github.com/datajoint/datajoint-python/issues/436.  The code above will look as:

The feature above will look as:
```
-> A
---
-> B.proj(x1='x', y1='y')
```

**Comment by austin-hilberg** [comment]
@dimitri-yatsenko so should this be labeled `wontfix`?

**Comment by dimitri-yatsenko** [comment]
Let's be sure to document the new syntax with good examples.

**Comment by dimitri-yatsenko** [comment]
The new syntax was implemented in datajoint-python 0.11.   The new syntax will be implemented in MATLAB in https://github.com/datajoint/datajoint-matlab/issues/140.  I am closing this issue because it is an implementation issue rather than documentation.
